components/item/MediaThumbnailGrid.vue

```diff
@@ -4,18 +4,21 @@
     class="d-flex flex-wrap mb-3"
   >
     <!-- TODO: populate alt, but with what? -->
-    <b-img-lazy
+    <button
```

Comment:
I do think it is valid HTML, an `<img>` inside a button, but could you make this a link? See also the notes on the description of the ticket.

Reply:
Now using anchor tags instead of buttons